packages/wdio-allure-reporter/src/index.js

```diff
@@ -138,22 +138,73 @@ class AllureReporter extends WDIOReporter {
     }
 
     onHookStart(hook) {
-        if (!this.allure.getCurrentSuite() || ignoredHooks(hook.title)) {
+        if (!this.allure.getCurrentSuite()) {
             return false
         }
 
-        this.allure.startCase(hook.title)
+        // todo: describe what are we doing here
```

Comment:
I suppose, it was backported from v4, as you I am no sure it is required or can be removed

Reply:
I'm not sure as well. I'd better keep existing logic as is for now.

We can improve it in different PR

Reply:
@BorisOsipov removed `// todo: describe what are we doing here` and refactored that code.